jenkins-pipelines/artifacts-centos7.jenkinsfile

```diff
@@ -9,6 +9,7 @@ artifactsPipeline(
     test_config: 'test-cases/artifacts/centos7.yaml',
     backend: 'gce',
     instance_provision: 'spot',
+    scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/branch-2.0/latest/scylla-manager.repo',
```

Comment:
TBH, I'm not sure if we need to take latest "unstable" or latest "official" one....
on one hand, it would be better to detect the issues before they reach the official one, OTOH we won't test the official one this way...

Reply:
that's also what I thought about... Unstable will eventually become RC and stable, right? If so we don't miss any version.

Reply:
I think that the best way to solve this is that Releng will provide the repo URL when triggering release so the relevant version will be tested...

Reply:
meanwhile, let's merge this, so we will start testing at least unstable. @roydahan WUT?

Reply:
Yes, I also discussed it with Shlomi and sent it on email.
Let's proceed with this for now.